definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -3125,5 +3125,16 @@
     "PostConditionState": "*",
     "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/return-to-state-before-general-referral",
     "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "11/03/2021",
+    "CaseTypeID": "DIVORCE",
+    "ID": "judgeCostOrderDecision",
```

Comment:
Should this new event be behind a feature toggle, or is this good to release in CICD?